src/main/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionRequestController.java

```diff
@@ -24,6 +29,7 @@
 public class ObjectionRequestController {
```

Comment:
Not related to this PR but maybe we should consider changing the name of this controller to just 'ObjectionController'.